### PR TITLE
add com.microsoft.EdgeDev

### PR DESCRIPTION
**Pull request by eyenpi**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. [GitHub issue](https://github.com/MicrosoftEdge/Status/issues/704): As the developers are not interested in flatpak and we already have an Edge package in flathub, I just wanted to publish the Dev version of the Edge as a flatpak in flathub. 
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. A dconf patch and zypac are used like the stable Edge repo.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.microsoft.EdgeDev.desktop [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+Comment[ta]=இணையத்தை அணுகுதல்
+Comment[te]=ఇంటర్నెట్‌ను ఆక్సెస్ చెయ్యండి
+Comment[th]=เข้าถึงอินเทอร์เน็ต
+Comment[tr]=İnternet'e erişin
+Comment[uk]=Доступ до Інтернету
+Comment[vi]=Truy cập Internet
+Comment[zh_CN]=访问互联网
+Comment[zh_HK]=連線到網際網路
+Comment[zh_TW]=連線到網際網路
+Exec=/app/bin/edge %U
```
Comment (hfiguiere) [inline com.microsoft.EdgeDev.desktop:108]:
```suggestion
Exec=edge %U
```

not an absolute path

Comment (eyenpi) [reply com.microsoft.EdgeDev.desktop]: Thanks. Changed all the absolute paths.

**Comment by eyenpi** [comment]
bot, build com.microsoft.EdgeDev

File: .github/workflows/update.yaml
Comment (JakobDev) [inline .github/workflows/update.yaml:1]: You don't need this

**Comment by barthalion** [comment]
What's the practical use case for this? We allowed ChromeDev as it's maintained by the mastermind behind making Chromium-based browsers work with Flatpak in the first place, but here I'm not sure what value it brings.

**Comment by eyenpi** [comment]
> What's the practical use case for this? We allowed ChromeDev as it's maintained by the mastermind behind making Chromium-based browsers work with Flatpak in the first place, but here I'm not sure what value it brings.

I didn't know the reason for allowing ChromeDev. I use ChromeDev (flatpak from flathub) and EdgeDev (AUR) daily to check out new features, So I decided to add the EdgeDev to flathub too. There is no value the same as ChromeDev for this. Just a typical flatpak application to be used by people like me. Is there any condition not to add Dev packages such as this?

File: com.microsoft.EdgeDev.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 the Chromium authors, with modifications -->
+<component type="desktop">
+  <id>com.microsoft.EdgeDev</id>
+  <launchable type="desktop-id">com.microsoft.EdgeDev.desktop</launchable>
+  <name>Microsoft Edge (dev)</name>
```
Comment (barthalion) [inline com.microsoft.EdgeDev.metainfo.xml:6]: Can you make this "Microsoft Edge Dev" instead, also in the desktop file? Seems that's what MS uses on other channels.

Comment (eyenpi) [reply com.microsoft.EdgeDev.metainfo.xml:6]: It's what they use in their desktop file in the deb package. The flathub beta package is also called Microsoft Edge (beta). Do you think I should change it anyway?

**Comment by hfiguiere** [comment]
bot, build com.microsoft.EdgeDev

**Pull request closed without merging**